Kernel/Time/TimeManagement.cpp

```diff
@@ -437,6 +437,8 @@ bool TimeManagement::disable_profile_timer()
 void TimeManagement::update_time_page()
 {
     auto* page = time_page();
+    if (!page)
```

Comment:
This seems like a less better way to fix the problem that occur here. We could simply initialize and create the time page before initializing any hardware timers, right? I'd prefer that option instead :)

Reply:
Sure, that also works, will do so